cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -196,6 +196,12 @@
     # NOTE: at this stage the /snap directory is stable as we have called
     # pivot_root already.
 
+    # Support the overmount interface which enables a custom userspace
+    mount options=(ro bind) /snap/*/** -> /*/**,
+    # But no fiddling with snap paths
+    audit deny mount options=(ro bind) /*/** -> /snap/*/**,
```

Comment:
This will probably deny valid bind mounts allowed by other rules (content interface).